.github/workflows/ci.yml

```diff
@@ -19,3 +19,24 @@ jobs:
       - run: yarn install --frozen-lockfile
       - run: yarn build
       - run: yarn test
+
+  publish:
+    if: github.ref == 'refs/heads/main'
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # extract `engines.node` from package.json and save it to output
```

Comment:
for now it's probably ok

Reply:
I'd say it's better to grab from `package.json engines` since that's what's declaring our package constraint, whereas its use in `nvmrc` is just telling a tool (`nvm`) what to check out.